graphql-dgs-codegen-core/src/test/kotlin/com/netflix/graphql/dgs/codegen/CodeGenTest.kt

```diff
@@ -74,6 +74,43 @@ class CodeGenTest {
             )
     }
 
+    @Test
+    fun `When the schema is empty, there is no parsing error`() {
+        val schema = """"""
+
+        CodeGen(CodeGenConfig(schemas = setOf(schema), packageName = basePackageName)).generate()
+    }
+
+    @Test
+    fun `When the schema contains just whitespace, there is no parsing error`() {
+        val schema = """
+            
+            
+            
+            
+            
+            
+            
+                    
+                    
+                    
+        """
+
+        CodeGen(CodeGenConfig(schemas = setOf(schema), packageName = basePackageName)).generate()
+    }
+
+    @Test
+    fun `When the schema is just an opening bracket, an invalid syntax (EOF) error is thrown`() {
+        val schema = """{""".trimIndent()
+
+        Assertions.assertThatThrownBy {
+            CodeGen(CodeGenConfig(schemas = setOf(schema), packageName = basePackageName)).generate()
+        }.isInstanceOf(CodeGenSchemaParsingException::class.java)
+            .hasMessageContainingAll(
+                "Invalid Syntax : offending token '<EOF>' at line 1 column 2"
```

Comment:
I would remove the check for exact message..that's not relevant as long as you have detected the parsing exception. For example, the underlying parser coudl change the message later, and we wouldn't care as long as we get an error for this scenario.